### PR TITLE
Add home-manager service module

### LINKED ISSUE #159: Add home-manager module
Using https://github.com/srid/neuron/pull/662 as inspiration.

### PR DESCRIPTION
**Pull request by rvl**
Resolves #159

### CODE REVIEW

**Comment by srid** [comment]
Nice! Could you add a page under https://note.ema.srid.ca/tips stating the steps to take to use this (on a Linux machine)?

**Comment by rvl** [comment]
Sure, good idea. I have pushed `docs/tips/nix.md`.

File: docs/tips/nix.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+Emanote can be easily built using the Nix expressions provided in its source repo.
+
+You will need [Nix](https://nixos.org/download.html) version 2.4 or greater.
+
+## Installing with Nix Flakes
+
+This will provide the `emanote` command in your environment.
+
+```sh-session
+$ nix profile install github:srid/emanote
```
Comment (srid) [inline docs/tips/nix.md:12]: TIL `nix profile install`.  Since Nix 2.4 got released I should probably use this for README.md too.

**Review by srid — APPROVED** [review]
Great, thanks!

**Pull request merged**